lxd/instance_snapshot.go

```diff
@@ -641,6 +643,8 @@ func snapshotPost(s *state.State, r *http.Request, snapInst instance.Instance, c
 		return response.BadRequest(err)
 	}
 
+	instName, _, _ := api.GetParentAndSnapshotName(snapInst.Name())
```

Comment:
Let's call this parentName as most places using this function seem to use that.

Reply:
I was just looking for occurrences in the same file, fixed.